### ISSUE TITLE
Custom extension looking for wrong directory

### ISSUE THREAD

**Issue by Vad1mo** [docker-library/php#106]
I am trying to install some php plugins following the tutorial on https://github.com/docker-library/docs/tree/master/wordpress.

here is my example
`RUN apt-get update && apt-get install -y \
        libfreetype6-dev \
        libjpeg62-turbo-dev \
        libmcrypt-dev \
        libpng12-dev \
	libzip-dev \
    && docker-php-ext-install iconv mcrypt zip
`

For the plugins the grep is incorrect `grep: /usr/local/etc/php/conf.d/docker-php-ext-iconv.ini` the correct path however would be  `grep: /usr/local/etc/php/conf.d/ext-iconv.ini`

```
Installing shared extensions:     /usr/local/lib/php/extensions/no-debug-non-zts-20131226/
Installing header files:          /usr/local/include/php/
grep: /usr/local/etc/php/conf.d/docker-php-ext-iconv.ini: No such file or directory
find . -name \*.gcno -o -name \*.gcda | xargs rm -f
find . -name \*.lo -o -name \*.o | xargs rm -f
find . -name \*.la -o -name \*.a | xargs rm -f 
find . -name \*.so | xargs rm -f
find . -name .libs -a -type d|xargs rm -rf
rm -f libphp.la       modules/* libs/*
+ cd /usr/src/php/ext/mcrypt
+ phpize
```

Is there a workaround or can this error be ignored.

**Comment by Vad1mo** [comment]
IMHO this line is not correct
https://github.com/docker-library/php/blob/master/docker-php-ext-install#L44 and should be changed to 

`ini="/usr/local/etc/php/conf.d/ext-$ext.ini"`

**Comment by tianon** [comment]
If you pull the latest version of this image, this should be gone -- it was a bug in the script that was fixed by #104.
